WhisperAPI/WhisperAPI/Models/Queries/SearchQuery.cs

```diff
@@ -18,9 +18,6 @@ public enum MessageType
         [JsonProperty("query")]
         public string Query { get; set; }
 
-        [JsonProperty("filteredQuery")]
```

Comment:
good, i spoke with Alex on this. Since we remove it from here to separate the model we want in endpoints and the model we persist, we should also remove Relevant and put it in ContextItem